packages/build/src/cli.js

```diff
@@ -76,13 +73,25 @@ exports.parse = function parse(argv) {
     .parse(cliArgs);
 
   options.nodeArgs = nodeArgs;
+  options.production = true;
 
   return options;
 };
 
 exports.run = async options => {
-  build(options).run((err, stats) => {
+  process.env.NODE_ENV = "production";
+
+  const config = loadWebpackConfig(options);
+  const compiler = webpack(config);
+
+  compiler.hooks.run.tapAsync("@marko/build", (_, done) =>
+    config.forEach(({ output: { path } }) => rimraf(path, done))
+  );
+
+  compiler.run(async (err, stats) => {
```

Comment:
Two suggestions here:

1. I don't we should be mixing async/await and callbacks because if `buildStaticSite` fails, it will result in an unhandled promise rejection error. Consider refactoring to something like this:

```js
function runWebpack(compiler) {
  return new Promise((resolve, reject) => {
    compiler.run((err, stats) => err ? reject(err) : resolve(stats));
  });
}

exports.run = async (options) => {
  ...
  const stats = await runWebpack(compiler);

  if (options.static) {
    await buildStaticSite(options, stats);
  }
}
```

2. Currently the  `buildStaticSite` will be called if `options.static` is provided regardless of whether `compiler.run` errors. Is `stats` null in the error case?  I didn't look at the implementation of `buildStaticSite`, but wasn't sure if it can handle that case.

Reply:
This code shouldn't have been in here, it's from another wip branch.  Thanks for highlighting it.